.github/workflows/build_navitia_packages_for_dev_multi_distribution.yml

```
@@ -36,10 +36,17 @@ jobs:
         path: "navitia_${{matrix.distribution}}_packages.zip"
     - name: remove useless temporary files
       run: rm -rf ../navitia-*
-    - name: run artemis NG on https://jenkins-core.canaltp.fr/job/artemis_ng
-      if: ${{matrix.distribution}} == 'debian8'
-      run: http -v -f POST https://${{secrets.JENKINS_NG_TOKEN}}@jenkins-core.canaltp.fr/job/artemis_ng/buildWithParameters
     - name: slack notification (the job has failed)
       if: failure()
       run: |
-          echo '{"text":":warning: Github Actions: build_navitia_${{matrix.distribution}}_packages_for_dev failed (https://github.com/CanalTP/navitia/actions?query=workflow%3A%22Build+Navitia+Packages+For+Dev+Multi+Distributions%22)"}' | http --json POST ${{secrets.SLACK_NAVITIA_CORE_TEAM_URL}}
+          echo '{"text":":warning: Github Actions: build_navitia_${{matrix.distribution}}_packages_for_dev failed (https://github.com/CanalTP/navitia/actions?query=workflow%3A%22Build+Navitia+Packages+For+Dev+Multi+Distributions%22)"}' | http --json POST ${{secrets.SLACK_NAVITIA_CORE_TEAM_URL}
```

```suggestion
          echo '{"text":":warning: Github Actions: build_navitia_${{matrix.distribution}}_packages_for_dev failed (https://github.com/CanalTP/navitia/actions?query=workflow%3A%22Build+Navitia+Packages+For+Dev+Multi+Distributions%22)"}' | http --json POST ${{secrets.SLACK_NAVITIA_CORE_TEAM_URL}}
```
Because i don't find the last '}' after this line.

---

thanks !